cookstyle.yml

```diff
@@ -0,0 +1,48 @@
+AllCops:
+  TargetRubyVersion: 3.1.6
```

Comment:
Note that the cookbook will be run by chef and thus not use system ruby, so 3.1.6 here is quite high. AFAIK the latest Chef client version (18.5.0) uses ruby 3.1.4, while 17.9.46 (our current version) uses Ruby 3.0.3.

My preference (as Criteo is the main user of the cookbook, even though it is publicly available) would be to target Chef 17.9.46 here and ruby 3.0.3.

Note that cookstyle supports TargetChefVersion as well as ruby version, to only suggest fixes relevant to this Chef version:
AllCops:
  TargetRubyVersion: 3.0.3 # Matches Chef client 17.9.46
  TargetChefVersion: 17.9.46

Reply:
Done